webapp/app/containers/Widget/components/Workbench/ConfigSections/TableSection/index.tsx

```diff
@@ -453,6 +456,19 @@ export class TableSection extends React.PureComponent<ITableSectionProps, ITable
                   </Select>
                 </Col>}
             </Row>
+            {!withPaging ? null : <Row>
+              <Col span={12}>
+                <Checkbox checked={autoPlay} onChange={this.checkboxChange('autoPlay')}>自动换页</Checkbox>
+              </Col>
+              <Col span={12}>
+                <InputNumber
+                  placeholder={"毫秒"}
+                  min={500}
```

Comment:
Consider the situation when the Table data is slow-query, and in this case the next page data may not have the response data with the interval which set by user, and finally the page will always be pending with no data represent.

Reply:
There is no good way to solve this problem. At present, the user can only be prompted to set an experience value.